tophatmonocle/pysaml2
# [WEB-24961] Upgrading pysaml2 to v4.6.5

## DevQA
- install the library `python setup.py sdist` (may need to sudo, I did)
  - if this fails try this first `pip install --upgrade setuptools --user python`
- copy the tarball to the container `cp dist/pysaml2_tophat-4.6.3.post2.tar.gz ../THM/`
- exec into docker `tophat exec thm`
- uninstall pysaml2_tophat just to make sure it's cleared out `pip uninstall pysaml2_tophat`
- install the tarball `pip install pysaml2_tophat-4.6.3.post2.tar.gz`

- make sure you have both of these flippers global `use_tophat_saml2_org` and `use_tophat_saml2_no_org`
- perform an sso login
- verify that it all works
- verify no issues pop up in sentry
